.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @hadim @mattip
\ No newline at end of file
+* @hadim
```

Comment:
Did you intend to remove me? I don’t mind either way, just as long as it was intentional

Reply:
Oh my bad. That wasn't the goal at all! Will add you back.

Reply:
Done at https://github.com/conda-forge/blessed-feedstock/pull/25.

Sorry again